src/components/search/src/components/IconSplash.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconDefinition } from '@fortawesome/fontawesome-svg-core';
+import type { ComponentType } from 'react';
+
+import { NEUTRALS } from '../../../../colors';
+
+const FigureWrapper = styled.figure`
+  margin: 10px auto;
+  text-align: center;
+
+  figcaption {
+    color: ${NEUTRALS[1]};
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 22px;
+    margin: 0 auto;
+    text-align: center;
+    width: 100%;
+  }
+`;
+
+type Props = {
+  /** Caption that renders underneath the icon */
+  caption ? :string;
+  /** Either a FontAwesome IconDefinition or a render prop */
+  icon ? :IconDefinition | ComponentType<any>;
+  /** size string that is passed to the FontAwesomeIcon or icon render prop */
+  size ? :string;
```

Comment:
It'll be good to define the enum type for this.

Reply:
We should have a talk about how to property expose flow types and such so IDEs will auto-fill and stuff